src/test/java/infinityworks/com/iwtechtest/IwTechTestApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package infinityworks.com.iwtechtest;
```

Comment:
Can we remove all the tests? There were none, and no test directory, by design. We want candidates to notice that and create the directory themselves